src/components/Offers/New/form-components/Policies/PrivacyPolicyContent.js

```diff
@@ -0,0 +1,411 @@
+                    </Typography>
+                    <Typography paragraph={true}>
+                    Administered by: Us
+                    </Typography>
+                    <Typography paragraph={true}>
+                    Purpose: These Cookies identify if users have accepted the use of cookies on the Website.
+                    </Typography>
+                </li>
+                <li>
+                    <Typography className={ classes.bold } paragraph={true}>
+                    Functionality Cookies
+                    </Typography>
+                    <Typography paragraph={true}>
+                    Type: Persistent Cookies
+                    </Typography>
+                    <Typography paragraph={true}>
+                    Administered by: Us
+                    </Typography>
+                    <Typography paragraph={true}>
+                    Purpose: These Cookies allow us to remember choices You make when You use the Website, such as remembering your login details or language preference. The purpose of these Cookies is to provide You with a more personal experience and to avoid You having to re-enter your preferences every time You use the Website.
+                    </Typography>
+                </li>
+                <li>
+                    <Typography className={ classes.bold } paragraph={true}>
+                    Analytics Cookies
+                    </Typography>
+                    <Typography paragraph={true}>
+                    Type: Session Cookies
+                    </Typography>
+                    <Typography paragraph={true}>
+                    Administered by: Google
+                    </Typography>
+                    <Typography paragraph={true}>
+                    Purpose: We use a tool called “Google Analytics” to collect information about use of this site. Google Analytics collects information such as how often users visit this site, what pages they visit when they do so, and what other sites they used prior to coming to this site. We use the information we get from Google Analytics only to improve this site. Google Analytics collects only the IP address assigned to you on the date you visit this site, rather than your name or other identifying information. We do not combine the information collected through the use of Google Analytics with personally identifiable information. Although Google Analytics plants a permanent cookie on your web browser to identify you as a unique user the next time you visit this site, the cookie cannot be used by anyone but Google. Google’s ability to use and share information collected by Google Analytics about your visits to this site is restricted by the&nbsp;
+                        <a href="https://www.google.com/analytics/terms/" target="_blank" rel="noopener noreferrer">
+                        Google Analytics Terms of Use
+                        </a>
+                    &nbsp;(as amended for government websites) and the&nbsp;
```

Comment:
Try replacing this type of lines and see if this works
```suggestion
                    {" (as amended for government websites) and the "}
```

Reply:
@CarlosMealha I think this is the only thing missing

Reply:
@CarlosMealha I don't know if you saw this